src/app/settings.js

```diff
@@ -149,6 +149,7 @@ Settings.tmpLocation = path.join(os.tmpDir(), Settings.projectName);
 Settings.databaseLocation = path.join(data_path, 'data');
 Settings.deleteTmpOnClose = true;
 Settings.automaticUpdating = true;
+Settings.UpdateSeed = true;
```

Comment:
naming convention is bad, you should use seedUpdate or redistributeUpdater, but don't start with an uppercase :)